apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/client/FaucetButton.tsx

```diff
@@ -201,23 +200,14 @@ export function FaucetButton({
     );
   }
 
-  // Email verification is required to claim from the faucet
-  if (
-    !accountQuery.data.emailConfirmedAt &&
-    !accountQuery.data.unconfirmedEmail
-  ) {
+  if (!isOnboardingComplete(accountQuery.data)) {
     return (
-      <>
-        <Button
-          variant="outline"
-          className="!opacity-100 w-full"
-          onClick={() => setShowOnBoarding(true)}
-        >
+      <Button variant="outline" asChild>
+        {/* TODO - instead of opening in new tab - set the next search param and open in same tab */}
```

Comment:
why not just do this now? what's preventing this change?

Reply:
I am working on this - This will be included in this PR

Reply:
Added it, removed todo